src/Resources/views/storefront/layout/meta.html.twig

```diff
@@ -271,6 +274,25 @@
                 }, 1);
                 window.EnderecoIntegrator.configScannerStarted = true;
             }
+
+            function setInitialSubdivisionalCode(EAO, formElement, subdivisionCodeSelector) {
```

Comment:
This looks like a repetition of what syncValue (window.EnderecoIntegrator.subdivisionMappingReverse) should be doing.
Here is the definiton of syncValue: https://github.com/Endereco/js-sdk/blob/master/modules/components/base.js#L281

I'm not happy with the whole block, because it repeats a functionality that is already in place but is not working for some reason.

Reply:
In Shopware - input with country states codes selection is loaded in async way by `CountryStateSelectPlugin`, that's why most probably it's not working by syncValue function, but it got fixed when I setted up initially `EAO.subdivisionCode` by grabbing country code id from `data-initial-country-state-id` (this marker is used by  Shopware `CountryStateSelectPlugin`). Thats why I made that function to cover Shopware-specific case for country states codes input.

Reply:
is there a callback for when `CountryStateSelectPlugin` is finished? The way you do it with manually setting the value of subdivision is probably the way to provide a quick fix to the problem, but I fear there might be a race condition if the `setInitialSubdivisionalCode` function is called before the country states are set by shopware.

Reply:
@michal127 please add a mechanism (setInterval would be ok) to wait and make sure, that the select has the options set by CountryStateSelectPlugin.